dev-docs/source/ReleaseChecklist.rst

```diff
@@ -15,260 +15,327 @@ Roles
-*  Perform Manual testing following the instructions described
+*  Ensure that Manual testing begins. The instructions for Manual testing are found 
    `here <https://www.mantidproject.org/Unscripted_Manual_Testing>`__.
 
-Maintenance
------------
-*  Present to the whole development team the maintenance tasks for this release period.
-*  Emphasize the order of work priorities as noted in the task priorities throughout this checklist.
-   Maintenance tasks may need to be paused to work on tasks for the release.
+Tuesday, 4 days
+---------------
 
-Create the Release Branch (once most PR's are merged)
------------------------------------------------------
+*  It is likely that many changes have been made over the beta test period, therefore 
+   ensure the manual testing is redone following the instructions described `here 
+   <https://www.mantidproject.org/Unscripted_Manual_Testing>`__.
 
-*  Ensure the
-   `master build and system
-   test <https://builds.mantidproject.org/view/Master%20Pipeline/>`__
-   jobs have passed for all build environments for this release.
-*  Run
-   `open-release-testing <https://builds.mantidproject.org/view/All/job/open-release-testing/>`__
-   to create the release branch and prepare build jobs
-*  Check state of all open pull requests for this milestone and decide which should be kept for the release,
-   liaise with PM on this. Move any pull requests not targeted for release out of the milestone
-   and run `update-pr-base-branch.py <https://github.com/mantidproject/mantid/blob/master/tools/scripts/update-pr-base-branch.py>`__
-   to update the base branches of those pull requests.
-*  Inform other developers that release-next has been created by adapting/posting the following announcement:
+.. _release-editor-checklist:
 
-  .. code
+Release Editor Checklist
+########################
 
-  The release branch for <version>, called release-next, has now been created: https://github.com/mantidproject/mantid/tree/release-next.  If you've not worked with the release/master-branch workflow before then please take a moment to familiarise yourself with the process: https://developer.mantidproject.org/GitWorkflow.html#code-freeze. The part about ensuring new branches have the correct parent is the most important part (although this can be corrected afterwards). All branches and PRs that were created before this release branch was created are fine, as their history is the same as master.
+**Role**: Person responsible for editing the release notes and giving them a common 
+language, layout, and collecting images.
 
-*  Create a skeleton set of release notes on master for the next version using the `python helper tool <https://github.com/mantidproject/mantid/blob/master/tools/release_generator/release.py>`_ and open a pull request to put them on ``master``.
+Wednesday, 2 weeks & 3 days
+---------------------------
 
+*  Create issues for people to neaten up the release notes and add images etc.
```

Comment:
It might be worth adding something about ensuring an image for the release is found. This can be a "collage" of images if there is not a big headline feature.